tests/e2e/update-strategy/86-verify-env-vars-exist.yaml

```diff
@@ -0,0 +1,24 @@
+# (c) Copyright [2022] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verify the license exists in each of the pods
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: kubectl exec -t -n $NAMESPACE v-ks-1-update-strategy-sc1-0 -- sh -c 'test $TZ = CET'
+  - script: kubectl exec -t -n $NAMESPACE v-ks-1-update-strategy-sc1-0 -- sh -c 'test $VERTICA_COM_GIT_REF = 1fab313k'
+  - script: kubectl exec -t -n $NAMESPACE v-ks-1-update-strategy-sc1-1 -- sh -c 'test $TZ = CET'
+  - script: kubectl exec -t -n $NAMESPACE v-ks-1-update-strategy-sc1-1 -- sh -c 'test $VERTICA_COM_GIT_REF = 1fab313k'
+  - script: kubectl exec -t -n $NAMESPACE v-ks-1-update-strategy-sc1-2 -- sh -c 'test $TZ = CET'
```

Comment:
I noticed this step failed in the azb run.  It is probably timing related.  I wonder if the statefulset controller is going through another rolling update for some reason?  We should look into that some more.